Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.13.1
+
+#ENV GO111MODULE=on
+#ENV GOPROXY=https://proxy.golang.org
+#ENV GOPROXY=direct
+
+ADD go.mod /go/src/github.com/minio/m3/go.mod
+ADD go.sum /go/src/github.com/minio/m3/go.sum
+WORKDIR /go/src/github.com/minio/m3/
+# Get dependancies - will also be cached if we won't change mod/sum
+RUN go mod download
```

Comment:
We don't need to do this. This is automatic

Reply:
I know, but I wanted cached as a layer since during development we most likely will be building the docker and pushing it to k8s to test